feg/cloud/go/services/feg_relay/servicers/feg_to_gw_relay.go

```diff
@@ -142,3 +141,20 @@ func getFegServedIds(networkId string) ([]string, error) {
 	}
 	return networkFegConfigs.ServedNetworkIds, nil
 }
+
+func validateFegContext(ctx context.Context) error {
+	fegId := protos.GetClientGateway(ctx)
+	if fegId == nil {
+		ctxMetadata, _ := metadata.FromIncomingContext(ctx)
+		errorStr := fmt.Sprintf(
+			"Failed to get Identity of calling Federated Gateway from CTX Metadata: %+v",
+			ctxMetadata,
+		)
+		glog.Error(errorStr)
+		return fmt.Errorf(errorStr)
+	}
+	if !fegId.Registered() {
+		return fmt.Errorf("federated gateway not registered")
+	}
+	return nil
+}
```

Comment:
just moved this function to the bottom

Reply:
Maybe it's worth moving it back up, pretty hard to see changes or lack of them like this.